include/tscore/Filenames.h

```diff
@@ -25,6 +25,19 @@
 // Configuration file names
 extern const char *STORAGE_CONF_FILENAME;
 extern const char *RECORDS_CONF_FILENAME;
+extern const char *VOLUME_CONF_FILENAME;
+extern const char *PLUGIN_CONF_FILENAME;
+
+extern const char *LOGGING_CONF_FILENAME;
+extern const char *CACHE_CONF_FILENAME;
+extern const char *IPALLOW_CONF_FILENAME;
+extern const char *HOSTING_CONF_FILENAME;
+extern const char *SOCKS_CONF_FILENAME;
+extern const char *PARENT_CONF_FILENAME;
+extern const char *REMAP_CONF_FILENAME;
+extern const char *MULTICERT_CONF_FILENAME;
+extern const char *SPLITDNS_CONF_FILENAME;
+extern const char *SNI_CONF_FILENAME;
```

Comment:
As long as you're doing this should we maybe switch to:
```
namespace CONF_FILENAME
{
extern const char * const STORAGE;
...
```

Reply:
Yeh, that makes sense, since we took the plunge on name spaces. Thanks Walt, I don't care what amc says about you, you have your moments.

Reply:
Sure, if I' ever get a ranch I'll name a pony after you.

Reply:
Or a mule, as in "I beat Zwoop like a rented mule".